index.html

```diff
@@ -62,82 +74,100 @@ <h3 class="project_name">
             Multi-Post
             Stories
         </h3>
 
         <ul class="first_list">
-            <li>CANOPY</li>
-            <li>Back End Dev</li>
+            <li>FACEBOOK</li>
+            <li>Full stack dev</li>
             <li>2015</li>
             
         </ul>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <p class="2nd_para">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
     <ul class="tech_list">
         <li>html</li>
         <li>css</li>
         <li>javaScript</li>
     </ul>
     <button class="btn1" type="submit">See Project</button>
 </div>
+</div>
 
 <div class="work_card">
-       
-    <img src="./images/thirdimage.png" alt="card"/>
+    <div class="whole_backspace mobile-nav">
+        <img src="./images/thirdimage.png" alt="card"/>
+    </div>
+    <div class="whole_backspace desk-nav">
+        <img src="./images/card_3.png" alt="icon"/>
+    </div>
+    <div class="side_content">
     <h3 class="project_name">
-        Multi-Post
-        Stories
+        FACEBOOK 360
     </h3>
 
     <ul class="first_list">
-        <li>CANOPY</li>
-        <li>Back End Dev</li>
+        <li>FACEBOOK</li>
+        <li>full stack Dev</li>
         <li>2015</li>
         
     </ul>
-    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <p>Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
 <ul class="tech_list">
     <li>html</li>
     <li>css</li>
     <li>javaScript</li>
 </ul>
 <button class="btn1" type="submit">See Project</button>
 </div>
+</div>
 <div class="work_card">
-       
-    <img src="./images/fourthimage.png" alt="card"/>
+    <div class="whole_backspace mobile-nav">
+        <img src="./images/thirdimage.png" alt="card"/>
+    </div>
+    <div class="whole_backspace desk-nav">
+        <img src="./images/card_4.png" alt="icon"/>
+    </div>
+    <div class="side_content">
     <h3 class="project_name">
-        Multi-Post
-        Stories
+        Uber Navigation
     </h3>
 
     <ul class="first_list">
-        <li>CANOPY</li>
-        <li>Back End Dev</li>
+        <li>Uber</li>
+        <li>Lead developer</li>
         <li>2015</li>
         
     </ul>
-    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <p>A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
 <ul class="tech_list">
     <li>html</li>
     <li>css</li>
     <li>javaScript</li>
 </ul>
 <button class="btn1" type="submit">See Project</button>
 </div>
-
+</div>
+</div>
   
 
-<section>
-    <div class="home_container">
-    <h1>About<br>Myself</h1>
+<section class="language_section">
+    <div class="home_container_forabout">
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_24](https://user-images.githubusercontent.com/69638013/160253852-34316d4b-adf8-4675-8811-fca2c8d4e98e.png)  | ![Screenshot_25](https://user-images.githubusercontent.com/69638013/160253848-6cac041c-34f2-4890-816f-6d9f902a1629.png)

### OPTIONAL

- [ ] kindly make the left block the div with class name `about_container` 40% and the right block the div with class name `language_container` 60% , please check the screenshot above, this will make your design look like the figma design 👍🏿 

- [ ] Kindly add the right padding to the background for each languages li text 👍🏿